src/components/activity/FormInput.js

```diff
@@ -0,0 +1,49 @@
+import {useState} from 'react';
+import {FormStyles} from './styles/FormInput.styles';
+import {nanoid} from 'nanoid';
+
+function FormInput({addActivity}) {
+  const [activity, setActivity] = useState({
+    id: nanoid(),
+    activity: '',
+    startTime: '00:00',
+    endTime: '00:00',
+  });
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    console.log(event);
+    addActivity(activity);
+    setActivity({id: nanoid(), activity: '', startTime: '00:00', endTime: '00:00'});
+  }
+
+  const handleChange = event => {
+    setActivity({...activity, [event.target.name]: event.target.value});
+  };
+
+  return (
+    <FormStyles onSubmit={handleSubmit}>
+      <label>
+        Aktivität hinzufügen:
+        <input type="text" name="activity" value={activity.activity} onChange={handleChange} />
```

Comment:
I think it is a good idea to add `required`. Otherwise you can set activities without name :) 
```suggestion
        <input type="text" name="activity" value={activity.activityName} onChange={handleChange} required/>
```